ideascube/library/views.py

```diff
@@ -13,10 +13,10 @@
 from django.views.generic import (CreateView, DeleteView, DetailView, FormView,
                                   ListView, UpdateView, View)
 
+from ideascube.configuration.models import Configuration
```

Comment:
Importing from configuration.**models** hints the user that Configuration is a model and will be use as it (query, create instance, ...). However, we do not use Configuration this way (even if in the implementation, it is a model)

Maybe use a wrapper or simply make the Configuration class directly importable from ideascube.configuration ?